CHANGELOG.md

```diff
@@ -1,6 +1,12 @@
 # Changelog
 
 All notable changes to this project will be documented in this file.
+### 0.1.9 ()
+
+### Bug Fixes
+
+* Added compatibility with cvl 1 ([#105](https://github.com/Certora/vscode-certora-prover/pull/105))
```

Comment:
Added dual support for both CVL versions (CVL1+CVL2), according to the installed certora-cli version.